include/ringmesh/mesh/mesh.h

```diff
@@ -35,15 +35,15 @@
 
 #pragma once
 
-#include <ringmesh/mesh/common.h>
-
 #include <algorithm>
 #include <memory>
+#include <ringmesh/mesh/common.h>

```

Comment:
Group all the includes of the same folder together (all the ringmesh/mesh/... includes one on top of the other, in alphabetical order of the file name).